tools/deb/BUILD

```diff
@@ -0,0 +1,51 @@
+load("@bazel_tools//tools/build_defs/pkg:pkg.bzl", "pkg_tar", "pkg_deb")
+
+pkg_tar(
+    name = "envoy-bin",
+    srcs = [
+        "istio-iptables.sh",
+        "istio-start.sh",
+        "//src/envoy/mixer:envoy",
+    ],
+    mode = "0755",
+    package_dir = "/usr/local/bin",
```

Comment:
See https://wiki.debian.org/FilesystemHierarchyStandard
`/opt` is for "Pre-compiled, non ".deb" binary distribution (tar'ed..) goes here."

https://wiki.debian.org/HowToPackageForDebian clearly says "DO NOT INSTALL ANY FILES" under `/usr/local`

For now we can do `/usr/local` if it involve many components, but eventually this should go to `/usr`.